docs/spec_checklist.md

```diff
@@ -19,7 +19,11 @@ reviews:
       1. Matches the ODS.
       1. Matches
          [StablehloOps.cpp](https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/StablehloOps.cpp).
-      1. If there are any mismatches, check that there are corresponding tickets.
+      1. If there are any mismatches, check that there are corresponding
+         tickets. If the ticket happens to be a generic one, applied to multiple
+         similar mismatches, then make sure to add a link of the ticket in all
+         the affected PRs: to keep track of all the PRs linked to the common
+         ticket.
```

Comment:
Let's perhaps mention all tickets (not just generic ones) in spec.md? That'll probably be the best way to make sure we don't forget anything.